packages/babel-plugin-transform-classes/test/fixtures/spec/computed-methods/output.js

```diff
@@ -8,9 +8,6 @@ var Foo = /*#__PURE__*/function () {
   babelHelpers.createClass(Foo, [{
     key: "foo",
     value: function foo() {}
-  }, {
-    key: "foo",
-    value: function foo() {}
```

Comment:
nit: Since now they can be merged, which is good. Consider revise the input to
```js
class Foo {
  foo() {}
  "foo"() { "string" }
  [bar]() {}
  [bar + "foo"]() {}
}
```
So we are sure that the latter overwrites the former, but not vice versa.